src/redux/actionTypes.ts

```diff
@@ -79,6 +79,9 @@ export enum ActionTypes {
   COLONY_ACTION_UNLOCK_TOKEN = 'COLONY_ACTION_UNLOCK_TOKEN',
   COLONY_ACTION_UNLOCK_TOKEN_ERROR = 'COLONY_ACTION_UNLOCK_TOKEN_ERROR',
   COLONY_ACTION_UNLOCK_TOKEN_SUCCESS = 'COLONY_ACTION_UNLOCK_TOKEN_SUCCESS',
+  COLONY_MOTION_MINT_TOKENS = 'COLONY_MOTION_MINT_TOKENS',
+  COLONY_MOTION_MINT_TOKENS_ERROR = 'COLONY_MOTION_MINT_TOKENS_ERROR',
+  COLONY_MOTION_MINT_TOKENS_SUCCESS = 'COLONY_MOTION_MINT_TOKENS_SUCCESS',
```

Comment:
Why did you add these here? They are not related to this PR.

Also, I think @chinins already added the proper ones in #2418

Reply:
I did indeed, @alicjakujawa you can cherry-pick my commits if you need to.